docs/docker-hub/builds.md

```diff
@@ -29,6 +29,8 @@ image was built exactly as specified.
 your repository on the Docker Hub registry.
 * Because the process is automated, Automated Builds help to
 make sure that your repository is always up to date.
+* Save network bandwidth and time pushing local Docker image 
```

Comment:
My take on this:

`You save network bandwidth and time, not having to push local Docker images to Docker Hub.`